packages/next/build/index.ts

```diff
@@ -1,5 +1,5 @@
 import crypto from 'crypto'
-import fs from 'fs'
+import { promises, writeFileSync } from 'fs'
```

Comment:
I'd alias the import as `fs` to avoid  this looking like a `promises` library
```suggestion
import { promises as fs, writeFileSync } from 'fs'
```